marimo/_plugins/ui/_impl/input.py

```diff
@@ -1308,6 +1308,22 @@ def __init__(
             Callable[[Sequence[FileUploadResults]], None]
         ] = None,
     ) -> None:
+        # Validate filetypes have leading dots
+        if filetypes is not None:
+            invalid_types = [
+                ft
+                for ft in filetypes
+                if not (
+                    ft.startswith(".")
+                    or ft in ("audio/*", "video/*", "image/*")
```

Comment:
we should allow more than these file types. just check if there is a `/` instead